packages/@aws-cdk/aws-ecs-patterns/README.md

```diff
@@ -294,7 +294,7 @@ const ecsScheduledTask = new ScheduledEc2Task(stack, 'ScheduledTask', {
     memoryLimitMiB: 256,
     environment: { name: 'TRIGGER', value: 'CloudWatch Events' },
   },
-  schedule: events.Schedule.expression('rate(1 minute)')
+  schedule: applicationautoscaling.Schedule.expression('rate(1 minute)')
```

Comment:
are the unit tests incorrect? they seem to also use `events` - https://github.com/aws/aws-cdk/blob/master/packages/%40aws-cdk/aws-ecs-patterns/test/ec2/test.scheduled-ecs-task.ts#L25

i'm wondering if the import from `autoscaling` is the error that needs to be fixed. The [doc string](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-ecs-patterns.ScheduledTaskBaseProps.html) for the `schedule` property also indicates that it's intentions are to use the `events module

Reply:
I have checked the git history, and noticed that the code was changed but the example was not updated. But looking at it again, it seems that the code was changed by mistake. What's triggering the task is a CloudWatch event, so the code should use events.Schedule.